2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_setup.py:_flush():75] Loading settings from /home/dvir/.config/wandb/settings
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_setup.py:_flush():75] Loading settings from /home/dvir/CLIP_Seg/Pytorch-UNet/wandb/settings
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Pytorch-UNet/train.py', 'program': 'train.py'}
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_setup.py:_flush():75] Applying login settings: {'anonymous': 'must'}
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_init.py:_log_setup():386] Logging user logs to /home/dvir/CLIP_Seg/Pytorch-UNet/wandb/run-20220228_165740-2m0iuhzv/logs/debug.log
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_init.py:_log_setup():387] Logging internal logs to /home/dvir/CLIP_Seg/Pytorch-UNet/wandb/run-20220228_165740-2m0iuhzv/logs/debug-internal.log
2022-02-28 16:57:40,839 INFO    MainThread:26498 [wandb_init.py:init():420] calling init triggers
2022-02-28 16:57:40,840 INFO    MainThread:26498 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-02-28 16:57:40,840 INFO    MainThread:26498 [wandb_init.py:init():471] starting backend
2022-02-28 16:57:40,840 INFO    MainThread:26498 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 16:57:40,881 INFO    MainThread:26498 [backend.py:ensure_launched():219] starting backend process...
2022-02-28 16:57:40,915 INFO    MainThread:26498 [backend.py:ensure_launched():224] started backend process with pid: 26731
2022-02-28 16:57:40,917 INFO    MainThread:26498 [wandb_init.py:init():480] backend started and connected
2022-02-28 16:57:40,920 INFO    MainThread:26498 [wandb_init.py:init():550] updated telemetry
2022-02-28 16:57:41,005 INFO    MainThread:26498 [wandb_init.py:init():581] communicating current version
2022-02-28 16:57:41,606 INFO    MainThread:26498 [wandb_init.py:init():586] got version response 
2022-02-28 16:57:41,607 INFO    MainThread:26498 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-28 16:57:42,294 INFO    MainThread:26498 [wandb_init.py:init():624] starting run threads in backend
2022-02-28 16:57:43,198 INFO    MainThread:26498 [wandb_run.py:_console_start():1827] atexit reg
2022-02-28 16:57:43,198 INFO    MainThread:26498 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-02-28 16:57:43,199 INFO    MainThread:26498 [wandb_run.py:_redirect():1706] Redirecting console.
2022-02-28 16:57:43,201 INFO    MainThread:26498 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-28 16:57:43,201 INFO    MainThread:26498 [wandb_init.py:init():651] run started, returning control to user process
2022-02-28 16:57:43,202 INFO    MainThread:26498 [wandb_run.py:_config_callback():966] config_cb None None {'epochs': 50, 'batch_size': 16, 'learning_rate': 2e-07, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2022-02-28 16:57:44,844 INFO    MainThread:26498 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-02-28 16:57:44,846 INFO    MainThread:26498 [wandb_run.py:_restore():1769] restore
2022-02-28 16:57:47,225 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 866
}

2022-02-28 16:57:47,623 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 866
}

2022-02-28 16:57:48,321 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 5880
}

2022-02-28 16:57:48,424 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 5880
}

2022-02-28 16:57:48,527 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 5880
}

2022-02-28 16:57:48,630 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 5880
}

2022-02-28 16:57:48,733 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 866
  total_bytes: 5880
}

2022-02-28 16:57:48,836 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3159
  total_bytes: 5880
}

2022-02-28 16:57:48,938 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:49,040 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:49,145 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:49,247 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:49,350 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:49,989 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}

2022-02-28 16:57:50,724 INFO    MainThread:26498 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5880
  total_bytes: 5880
}
local_info {
}

2022-02-28 16:57:51,813 INFO    MainThread:26498 [wandb_run.py:_append_files():2194] logging synced files
